Argo/Types/StandardTypes.swift

```diff
@@ -72,11 +157,46 @@ public extension CollectionType where Generator.Element: Decodable, Generator.El
   }
 }
 
+/**
+  Decode `JSON` into an array of values where the elements of the array are
+  `Decodable`.
+
+  If the `JSON` is an array of `JSON` objects, this returns a decoded array
+  of values by mapping the element's `decode` function over the `JSON` and
+  then `sequencing` the result. This makes this `Decode` function an
```

Comment:
Same here should `Decode` be `decode` or in this case maybe even `decodeArray`? The last option would also change the sentence to for example “This makes the `decodeArray` function...” or “This makes the function...”.